x/nexus/keeper/keeper_test.go

```diff
@@ -74,26 +71,14 @@ func TestLinkAddress(t *testing.T) {
 	repeats := 20
 
 	var ctx sdk.Context
+	cfg := app.MakeEncodingConfig()
+	keeper, ctx = setup(cfg)
```

Comment:
Won't this share the context across the tests/repetitions in this function? Doesn't seem to matter for this specific test though.